src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -440,6 +440,9 @@
   - name: --enable-encryption-at-host
     type: bool
     short-summary: Enable EncryptionAtHost, default value is false.
+  - name: --enable-azure-rbac
+    type: bool
+    short-summary: Enable Azure role assignments to control authorization checks on cluster.
```

Comment:
this should be role based access control or RBAC?

Reply:
Let's go with role assignment. At the end of the day the user expectation is to get them to use Azure role assignments to control k8s data plane authorization. So let's describe it that way itself

Reply:
Yes, it should be either Azure RBAC or Azure role based access control.
e.g. https://docs.microsoft.com/en-us/azure/role-based-access-control/overview

Reply:
Yes, I know it's Azure RBAC.... i'm saying for description let's just describe the user experience itself.... That one is using Azure role assignments to control authorization checks on cluster.